apps/www/app/glossary/[slug]/_faq.tsx

```
@@ -0,0 +1,38 @@
+      <div className="text-center space-y-4 pb-6 mx-auto">
+        <h3 className="not-prose blog-heading-gradient text-left font-medium tracking-tight text-4xl">
+          {props.title}
+        </h3>
+        <p className="font-medium leading-7 not-prose text-white/70 lg:text-xl text-sm md:text-base py-6 max-w-sm md:max-w-md lg:max-w-xl xl:max-w-4xl text-left">
+          {props.description}
+        </p>
+      </div>
+      <div className="mx-auto md:max-w-[800px]">
+        <Accordion type="single" collapsible className="w-full">
+          {props.items.map((item) => (
+            <AccordionItem value={item.question}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)